terraform/variables.tf

```diff
@@ -67,6 +68,24 @@ variable "object_storage" {
   }
   default = {}
 }
+variable "redis_params" {
+  type = object({
+    host          = optional(string, "redis")
+    exposed_port  = optional(number, 6380)
+    user          = optional(string, "admin")
+    password      = optional(string, "admin")
+    scheme        = optional(string, "redis")
+    tls_enabled   = optional(bool, true)
+    Ssl           = optional(bool, true)
+    ca_path       = optional(string, "/redis/certs/ca.pem")
+    cert_path     = optional(string, "/redis/certs/redis.crt")
+    key_path      = optional(string, "/redis/certs/redis.key")
+    timeout       = optional(string, "20000")
+    database_name = optional(string, "redis_database")
+    windows       = optional(bool, false)
+  })
+  default = {}
+}
```

Comment:
Do not bother with that, they are internal redis module internal parameters that do not need to be exposed.